frontend/public/components/marketplace/kubernetes-marketplace.jsx

```diff
@@ -0,0 +1,132 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import * as PropTypes from 'prop-types';
+import { Helmet } from 'react-helmet';
+
+import {Firehose, PageHeading, StatusBox} from '../utils';
+import {referenceForModel} from '../../module/k8s';
+import {normalizeIconClass} from '../catalog-item-icon';
+import {PackageManifestModel} from '../../models';
+import CatalogTileView from 'patternfly-react-extensions/dist/esm/components/CatalogTileView/CatalogTileView';
+import CatalogTile from 'patternfly-react-extensions/dist/esm/components/CatalogTile/CatalogTile';
+
+const normalizePackageManifests = (packageManifests, kind) => {
+  const activePackageManifests = _.filter(packageManifests, packageManifest => {
+    return !packageManifest.status.removedFromBrokerCatalog;
+  });
+  return _.map(activePackageManifests, packageManifest => {
+    const tileName = packageManifest.metadata.name;
+    const iconClass = 'fa fa-clone'; // TODO: get this info from the packagemanifest
+    const iconObj = _.get(packageManifest, 'status.channels[0].currentCSVDesc.icon[0]', null);
+    const tileImgUrl = iconObj ? `data:${_.get(iconObj, 'mediatype')};base64,${_.get(iconObj, 'base64data')}` : null;
+    const tileIconClass = tileImgUrl ? null : iconClass;
+    const tileDescription = packageManifest.metadata.description;
+    const tileProvider = packageManifest.metadata.labels.provider;
```

Comment:
`_.get` here just in case since labels can be empty in the API

```js
const tileProvider = _.get(packageManifest, 'metadata.labels.provider');
```